src/views/communities/Stories.tsx

```diff
@@ -109,23 +109,22 @@ export default function Stories() {
                 showsHorizontalScrollIndicator={false}
                 contentContainerStyle={{ padding: 18 }}
             >
-                {userAddress.length > 0 &&
+                {/* {userAddress.length > 0 &&
```

Comment:
Only logged users in a valid (aka accepted and public) community (beneficiary or managers) should be able to add stories. Why are you changing this?